composer.json

```diff
@@ -14,9 +14,10 @@
     "require": {
       "php": "7.*",
       "ext-json": "*",
-      "automattic/jetpack-connection": "1.30.13",
-      "automattic/jetpack-config": "1.5.3",
-      "automattic/jetpack-autoloader": "2.10.10",
+      "automattic/jetpack-connection": "1.36.0",
+      "automattic/jetpack-config": "1.6.0",
+      "automattic/jetpack-autoloader": "2.10.11",
+      "automattic/jetpack-identity-crisis": "0.6.x-dev",
```

Comment:
TODO: Jetpack team will release jetpack-identity-crisis 0.6.4 on Jan 25, 2022 (every Tuesday). Then I will switch it to this stable version.

Reply:
Addressed ab724f0d7ce4d56573e885282bdef0532cfc3b8a